maint-lib/stage.py

```diff
@@ -45,10 +45,26 @@
     sys.exit(1)
 
 
+def populateCephPointRelease():
+    # We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
+    # It could be under the two following syntax :
+    # - package_name-package_version
+    # - package_name=package_version"
+    os.environ["CEPH_POINT_RELEASE"] = ""
+    if os.environ["CEPH_VERSION"]:
+        for separator in ["-", "="]:
+            if separator in os.environ["CEPH_VERSION"]:
+                version, point_release = os.environ["CEPH_VERSION"].split(separator)
+                os.environ["CEPH_VERSION"] = version
+                os.environ["CEPH_POINT_RELEASE"] = "{}{}" . format(separator, point_release)
+
+
 def main(CORE_FILES_DIR, CEPH_RELEASES_DIR):
     logging.info('\n\n\n')  # Make it easier to determine where new runs start
     logging.info('Start time: {}'.format(time.ctime()))
 
+    populateCephPointRelease()
```

Comment:
This should go after `verifyRequiredEnvVars()`

Reply:
No as it does populate the C_P_R